Software/DumptruckUltra-MCU/hw/pressure_sensor/test/test_pressure_sensor.cpp

```diff
@@ -0,0 +1,17 @@
+#include "cyhal_psoc6_01_43_smt.h"
+#include "hw/pressure_sensor/pressure_sensor.hpp"
+#include "proc_setup.hpp"
+#include <iostream>
+
+using namespace std;
+
+int main() {
+    Hardware::Processor::setupProcessor();
+
+    using Hardware::Pressure_Sensor::Pressure;
+
+    // Setup ADC
+    Pressure adc(P10_0);
+    cout << "ADC Reading: " << adc.read();
```

Comment:
Requires `retarget_io` to set up.  You also probably want `"\r\n"` for readability.